theme.json

```diff
@@ -224,7 +236,16 @@
 			},
 			"core/post-title": {
 				"typography": {
-					"fontSize": "var(--wp--preset--font-size--xx-large)"
+					"fontSize": "var(--wp--preset--font-size--large)",
+					"letterSpacing": "-2px",
+					"fontWeight": "400"
+				},
```

Comment:
I believe I saw a discussion about changing the font weight site wide for the headings, but I don't recall where so I set it to 400 here to match the figma design.